docs/building-your-quest/index.md

```diff
@@ -37,27 +37,27 @@ quest.zip
 This is a mandatory file that must be placed at the root of the zip archive's structure. This file defines the metadata of the quest:
 
 ```yaml
-id: a unique quest identifier. Should not change. 3-50 characters, must begin with a letter. Accepted characters are A-Z, a-z, 0-9, and "_" | mandatory
-title: Quest title. 2-40 characters. All characters are accepted. "Funnel Drop" in the example below | mandatory
-subtitle: Quest subtitle. 2-68 characters. All characters are accepted. "Find the cause of user traffic drop in production and fix it" in the example below | mandatory
-description: Quest description. All characters are accepted. "What's worse.." in the example below | mandatory 
-summary: Quest summary. Shown when users finish the quest | mandatory 
-slogan: Quest slogan. 2-1000 characters. "It's time for.." in the example below | mandatory
-level: Quest level. beginner/intermediate/advanced | mandatory
-duration: Estimated number of hours required to finish the quest | mandatory
+*id: a unique quest identifier. Should not change. 3-50 characters, must begin with a letter. Accepted characters are A-Z, a-z, 0-9, and "_" | mandatory
```

Comment:
id*

Reply:
changed, but still doesn't like it. 
Maybe I'll try a table later

Reply:
I hate tables